packages/d/dpp/xmake.lua

```diff
@@ -53,7 +53,9 @@ package("dpp")
 
     add_deps("nlohmann_json", "openssl", "zlib")
 
-    add_configs("have_voice", { description = "Enable voice support for the library.", default = true, type = "boolean" , readonly = true})
+    add_configs("have_voice", { description = "Enable voice support for the library.", default = true, type = "boolean" , readonly = false})
```

Comment:
please rename `have_voice` to `voice`, we need not any have_  with_ enable_ preifx.

Reply:
[This was changed](https://github.com/xmake-io/xmake-repo/pull/4824/commits/84b45d9f44c4a238f933fab601f7e9764c7d5dcb) but the main issue is that this change __will break__ the current user code bases, the flag was left unchanged since 2022 ([introduced here](https://github.com/xmake-io/xmake-repo/pull/1596)), the most reliable solution I was considering is showing a deprecation warning to anyone is using `have_voice` flag and then in later DPP versions erase it from the next versions.

Another solution I propose is doing a temporal alias from `have_voice` to `voice` internally in the script, and doing the same as before, show a deprecation warning and then delete the flag.

Reply:
you can reserve have_voice and add some deprecated warnings.

and add new voice config , then tell user to use voice instead of have_voice

Reply:
Added in the last commit, please review it for other advices, thanks for the support!